extensions/adobe/experience/campaign-experienceevent.schema.json

```diff
@@ -17,91 +17,414 @@
+            },
+            "xdm:seedMember": {
+              "title": "Seed Member",
+              "type": "boolean",
+              "default": false,
+              "description": "The recipient of this message is a member of a seed list for this campaign activity. Seed addresses are used to include recipients who do not match the defined target criteria. Seed members could be your marketing team or other stakeholders who are interested in tracking/measuring the quality of the campaign. SeedMembers get the actual message, at the same time with regular recipients, even when the seedMember does not qualify to be a regular recipient."
+            },
+            "xdm:quarantine": {
+              "title": "Quarantine",
+              "type": "boolean",
+              "default": false,
+              "description": "Address moved to quarantine. The quarantine is set to `true` when the message failed to deliver in a way that indicates that no future messages will be deliverable either to this address."
+            },
+            "xdm:proofMember": {
+              "title": "Proof Member",
+              "type": "boolean",
+              "description": "The recipient of this message is a member of the proof group for this campaign activity. The proof is a special message that lets you test a delivery before sending to the main target. Proof recipients are responsible for approving both the form and content of the message. It is a pre-delivery target.",
+              "default": false
+            },
+            "xdm:controlGroupMember": {
+              "title": "Control Group Member",
+              "type": "boolean",
+              "description": "The recipient of this message is a member of a control group for this campaign activity. A control group is a population which will not receive the delivery; it is used to track post-delivery behavior and campaign impact by making a comparison between the control group (or hold out group) with the behavior of target population, which has received the delivery.",
+              "default": false
+            },
+            "xdm:testMember": {
+              "title": "A/B Test Member",
+              "type": "boolean",
+              "description": "The recipient of this message is a member of a test group for this campaign activity. This is primarily used for identifying the initial population of an A/B Test. After sending different variants to the testMembers, the winner Variant is decided to be sent to the remaining population based on behavioural changes in testMembers.",
+              "default": false
+            },
+            "xdm:size": {
```

Comment:
we dont use this today. Since the aim is to make the schema compact, we can remote this and re-introduce this whenever we start sending this detail

Reply:
@saurabhere what don't you use?  xdm:size isn't used?  Sorry wasn't clear from the comment.

Reply:
Yes `xdm:size` isn't used today.